docs/en/features/completion.md

```
@@ -0,0 +1,92 @@
+
+## Changelog
+
+| Date | Change | PR |
+|------|--------|-----|
+| — | Initial include/semantic completion | — |
+| — | Module import completion (flat prefix) | — |
+| 2025-06 | Space trigger + extension middleware gate | [#460](https://github.com/clice-project/clice/pull/460) |
+
+[^clangd-2622]: [clangd#2622](https://github.com/clangd/clangd/issues/2622) — Remove other modules' partitions from suggestions `open`
+[^clangd-2623]: [clangd#2623](https://github.com/clangd/clangd/issues/2623) — Fix missing icons for module import suggestions `closed`
+[^clangd-2626]: [clangd#2626](https://github.com/clangd/clangd/issues/2626) — Single-token completion for `import`/`module` keywords `open`
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "^clangd-2626"

(MD053, link-image-reference-definitions)